packages/core/src/package/packageInstallers/InstallSourcePackageImpl.ts

```diff
@@ -290,18 +289,26 @@ export default class InstallSourcePackageImpl extends InstallPackage {
       await reconcileProfileAgainstOrg.exec();
 
       //Now deploy the profies alone
-      fs.appendFileSync(
-        path.join(sourceDirectoryPath, ".forceignore"),
-        "**.**" + os.EOL
-      );
-      fs.appendFileSync(
-        path.join(sourceDirectoryPath, ".forceignore"),
-        "!**.profile-meta.xml"
-      );
+
+      const profilesDirs = glob.sync("**/profiles/", {
+        cwd: path.join(sourceDirectoryPath, sourceDirectory),
+        absolute: true
+      });
+
+      const profileDeploymentStagingDirectory = path.join(sourceDirectoryPath, "ProfileDeploymentStagingDirectory");
+      fs.mkdirpSync(path.join(profileDeploymentStagingDirectory, sourceDirectory, "profiles"));
+
+      for (const dir of profilesDirs) {
+        // Duplicate profiles are overwritten
+        fs.copySync(dir, path.join(profileDeploymentStagingDirectory, sourceDirectory, "profiles"))
+      }
+
+      fs.copySync(path.join(sourceDirectoryPath, "sfdx-project.json"), path.join(profileDeploymentStagingDirectory, "sfdx-project.json"));
+      fs.copySync(path.join(sourceDirectoryPath, ".forceignore"), path.join(profileDeploymentStagingDirectory, ".forceignore"));
```

Comment:
is the forceignore at this part takes multiple ignores into consideration or is it handled already before it?

Reply:
Another aspect, that we should do is we should do double reconcile only if the package has payloads other than profiles